schemas/config.json

```diff
@@ -0,0 +1,397 @@
+                                "recreate",
+                                "fail",
+                                "ask"
+                            ]
+                        },
+                        "destination": {
+                            "title": "Destination",
+                            "description": "Where models yml files will be generated\nYou can specify a different path by declaring it explicitly, i.e.: \"models/staging/{{model_file_name}}.yml\"",
+                            "type": "string",
+                            "minLength": 1,
+                            "pattern": ".+({{model_file_name}}.yml)$"
+                        },
+                        "models": {
+                            "title": "Models",
+                            "description": "Model(s) path where 'generate properties' will look for models for generation",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "select": {
+                            "title": "Select",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "exclude": {
+                            "title": "Exclude",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "selector": {
+                            "title": "Selector",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        }
+                    },
+                    "required": []
+                },
+                "metadata": {
+                    "title": "Metadata",
+                    "description": "Generate metadata parameters",
+                    "type": "object",
+                    "properties": {
+                        "database": {
+                            "title": "Database",
+                            "description": "Database where to look for source tables",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "schemas": {
+                            "title": "Schemas",
+                            "description": "List of schema names where to look for source tables",
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                                "minLength": 1
+                            },
+                            "minItems": 1
+                        },
+                        "relations": {
+                            "title": "Relations",
+                            "description": "",
```

Comment:
Default relations to filter when generating metadata file (not recommended as a config option)